### PR TITLE
Login checker

### PR DESCRIPTION
**Pull request by bhrg3se**
Added login check function per page rather than hardcoding same logic for everything.

### CODE REVIEW

**Comment by bhrg3se** [comment]
closes #28

File: mobileapp/src/selectors/facebook/AdPreferences.tsx [hunk 1]
```diff
@@ -31,12 +30,14 @@ const extractAdsBasedonRelationshipStatus = (htmlContent: string) => {
...
 
-    const selected = $(
-      '#root > div:nth-child(1) > div > article > div:nth-child(2) > div:nth-child(1) > div > div > div:nth-child(2) > div > div',
-    );
+    const selected = $('input[id="u_0_3l"]');
 
-    // resolve('ok');
-    resolve(selected.prop('aria-checked'));
+    const checked = selected.prop('checked');
+    if (checked == '1') {
```
Comment (sshahcodes) [inline mobileapp/src/selectors/facebook/AdPreferences.tsx:36]: why tripel equality check is not used here?

Comment (bhrg3se) [reply mobileapp/src/selectors/facebook/AdPreferences.tsx:36]: selectors/facebook/* is still work in progress.

**Review by sshahcodes — COMMENTED** [review]
- Remove commented code which are not required for reference. 

codes such as below for reference:
1. ``` isLoggedIn:`var isLoggedIn=document.querySelectorAll('a[href^="https://accounts.google.com/ServiceLogin"]').length===0;` 

2. ```pageURL: 'https://www.linkedin.com/mypreferences/m/settings/ads-personalization',```

3. ``` fixURL: 'https://www.linkedin.com/mypreferences/m/settings/manage-my-active-status'```

 are hard copied and used in various source file. Better define it once and then call that instance in other source files.

**Review by sshahcodes — APPROVED** [review]

**Pull request merged**